packages/plugin-ext/src/common/uri-components.ts

```diff
@@ -11,4 +11,11 @@ export interface UriComponents {
     path: string;
     query: string;
     fragment: string;
+    external?: string;
+}
+
+// some well known URI schemas
+export namespace Schemes {
+    export const file = 'file';
```

Comment:
const should be uppercase ?

Reply:
I haven't find any code style rules about constants, so yes I rename it to `File`

Reply:
ok I thought FILE was for a constant, File a variable